src/VisualStudio/CSharp/Impl/Options/Formatting/CodeStylePage.cs

```diff
@@ -433,11 +433,13 @@ private static string CSharpCodeStyleOptions_GenerateEditorconfig(OptionSet opti
             return editorconfig;
         }
 
-        private static string CSharpCodeStyleOptions_GenerateEditorconfig(OptionSet optionSet, PerLanguageOption<CodeStyleOption<bool>> option)
+        private static void DotNetCodeStyleOptions_GenerateEditorconfig(OptionSet optionSet, PerLanguageOption<CodeStyleOption<bool>> option, StringBuilder editorconfig)
         {
+            editorconfig.Append(((EditorConfigStorageLocation<CodeStyleOption<bool>>) option.StorageLocations[0]).KeyName);
```

Comment:
note: this is slightly unsafe, because it presumes the EditorConfigStorageLocation will be in the 0th location in the array.  better would be to actually find the right item in that array in case it's not the first.